internal/scheduleringester/schedulerdb.go

```diff
@@ -81,6 +83,52 @@ func (s *SchedulerDb) WriteDbOp(ctx context.Context, op DbOperation) error {
 				return errors.WithStack(err)
 			}
 		}
+	case UpdateJobSchedulingInfo:
+		args := make([]interface{}, 0, len(o)*3)
+		argMarkers := make([]string, 0, len(o))
+
+		currentIndex := 1
+		for key, value := range o {
```

Comment:
this might be simpler as a `pgx.batch` query (https://github.com/jackc/pgx/blob/master/batch_test.go#L30). I suspect the performance won't be much worse either.